pyproject.toml

```diff
@@ -3,3 +3,6 @@
 [build-system]
 requires = ["setuptools>=34.4", "wheel", "setuptools_scm"]
 build-backend = "setuptools.build_meta"
+
+[tool.usort]
+first_party_detection = false
```

Comment:
This is the setting also in fb-internal repos, making this change for consistency.